src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/InsertOneTableIntegrationTest.java

```diff
@@ -173,6 +181,107 @@ public final void createDefaultTables() {
                 Map.of("type", "vector", "valueType", "float", "dimension", 3)),
             "id")
         .wasSuccessful();
+
+    assertNamespaceCommand(keyspaceName)
```

Comment:
Instead of adding every table in init block, tables that are only used by a single test method could (and I think should) be moved to that test method: first create table, then try insert which succeeds or fails.
And only use init method for tables shared across multiple tests.

Doing this keeps table and usage close to each other and avoids humongous methods.